src/components/ConnectProjectSelector/ConnectProjectSelector.vue

```diff
@@ -103,13 +116,25 @@
           <unnnic-skeleton-loading :style="{ marginBottom: '16px' }" tag="div" height="50px" />
           <unnnic-skeleton-loading tag="div" width="100%" height="50px" />
         </div>
+
+        <div
+          v-else-if="
+            projectsSearch
+            && projects[currentOrgUuid].status === 'complete'
+            && projectsFiltered.length === 0
```

Comment:
You can transform this validation into a computed variable to make the html section more readable